pkg/secrets/secrets.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2023 Adevinta
+*/
+
+package secrets
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	vterrors "github.com/adevinta/vulcan-tracker/pkg/errors"
+	"github.com/labstack/echo/v4"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	awsconfig "github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/secretsmanager"
+)
+
+const region = "eu-west-1"
+
+type (
+	// Secrets manage the credentials of ticket trackers servers.
+	Secrets interface {
+		GetServerCredentials(serverID string) (Credentials, error)
+	}
+
+	// AWSSecrets represents the specific AWS secret manager.
+	AWSSecrets struct {
+		Logger echo.Logger
+	}
+
+	// Credentials store the credentials of a secret.
+	Credentials struct {
+		User     string `json:"user"`
+		Password string `json:"password"`
+	}
+)
+
+// GetServerCredentials return the user and password inside a Credentials type
+// from AWS secret manager for a specific server.
+func (s *AWSSecrets) GetServerCredentials(serverID string) (Credentials, error) {
+
+	secretName := fmt.Sprintf("/vulcan/k8s/tracker/jira/%s", serverID)
+	config, err := awsconfig.LoadDefaultConfig(context.TODO(), awsconfig.WithRegion(region))
```

Comment:
I would't fix the region here as some users of the component may want to use a different one, what we could do is to check this [env var](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-the-aws-region) and it is not set then set by default the region to `eu-west-1`

Reply:
Fixed in commit ce9cece908f3f3ed5744069c61cfd9540f824107